addons/mail/static/src/messaging/component/thread_preview/thread_preview.scss

```diff
@@ -6,7 +6,7 @@
     @include o-mail-notification-list-item-layout();
 
     &:hover .o_ThreadPreview_markAsRead {
-        display: flex;
+        opacity: 1;
```

Comment:
should be style category, and specificity issue here so hover should be handle in OWL instead of pure css.

Reply:
Will be done in task-2258605